src/equations/compressible_navier_stokes_2d.jl

```diff
@@ -0,0 +1,330 @@
+@doc raw"""
+    CompressibleNavierStokesEquations2D(gamma, Re, Pr, Ma_inf, equations)
```

Comment:
Looking at this after quite some time again, I feel like the naming is not optimal. We now have "SomethingEquations", which are really complete equations (like Euler, advection, MHD etc.), and we have "SomethingOtherEquations", which are just stubs. Maybe we should name these something like `CompressibleNavierStokresDiffusion2D" to avoid confusing new users?

This has not to be resolved in this PR but maybe added to the meta issue (#1147 ?) to be discussed at a later stage? Thus feel free to resolve if it is recorded somewhere such that we don't forget.

Reply:
I agree that we should discuss the naming of this stuff again. Your suggestion
```julia
CompressibleNavierStokesEquations2D = (CompressibleEulerEquations2D, CompressibleNavierStokesDiffusion2D)
```
sounds reasonable to me.

Reply:
I like that too. I'll make these changes soon.

Reply:
Done in https://github.com/trixi-framework/Trixi.jl/commit/4f757d303b7506b71014445a4c50747d396b916a